src/Executable.js

```diff
@@ -105,6 +111,80 @@ export default class Executable {
         return this;
     }
 
+    /**
+     * @param {number} minBackoff
+     * @returns {this}
+     */
+    setMinBackoff(minBackoff) {
+        if (minBackoff == null) {
+            throw new Error("minBackoff cannot be null.");
+        } else if (this._maxBackoff != null && minBackoff > this._maxBackoff) {
+            throw new Error("minBackoff cannot be larger than maxBackoff.");
+        }
+        this._minBackoff = minBackoff;
+        return this;
+    }
+
+    /**
+     * @returns {number | null}
+     */
+    get minBackoff() {
+        return this._minBackoff;
+    }
+
+    /**
+     * @param {?number} maxBackoff
+     * @returns {this}
+     */
+    setMaxBackoff(maxBackoff) {
+        if (maxBackoff == null) {
+            throw new Error("maxBackoff cannot be null.");
+        } else if (this._minBackoff != null && maxBackoff < this._minBackoff) {
+            throw new Error("maxBackoff cannot be smaller than minBackoff.");
+        }
+        this._maxBackoff = maxBackoff;
+        return this;
+    }
+
+    /**
+     * @returns {number | null}
+     */
+    get maxBackoff() {
+        return this._maxBackoff;
+    }
+
+    /**
+     * @private
+     * @param {?number} minBackoff
+     * @param {?number} maxBackoff
+     * @returns {this}
+     */
+    _setBackoff(minBackoff, maxBackoff) {
```

Comment:
Damn, I see why you added the `@ts-ignore` :( This is definitely my fault for forgoting that it would error from `tsc` instead of `eslint`. Unfortunately, I really don't want us to add `@ts-ignore` anywhere, so can we remove `[Executable|Client]._setBackoff()` and `[Executable|Client]._backoff()`. We might add those back later, but for this PR lets just keep it as in-sync with the other SDKs as we can.

Reply:
i swear